### ISSUE TITLE
Option to Publicly Share Quotes

### ISSUE THREAD

**Issue by TomatDividedBy0** [bookwyrm-social/bookwyrm#1275]
Part of the social aspect of GoodReads is being able to post and browse quotes publicly, which also helps those browsing the book to get an idea of whats inside. Combine this with federation and I think there's the potential to create a good system for aggregating quotes. IIRC mouse mentioned that this could raise some issues w/rt to harassment but I think having an option (off by default) to share quotes Private/Local/Federated would help solve that so that way people can pick the privacy setting relevant to them.

**Comment by realityfabric** [comment]
Is this separate from being able to share quotes to your feed and have followers like / boost / comment?

I think it would be really cool to have an aggregated list of quotes searchable by author / book.

Since we can post quotes with various privacy levels, would it make sense only to include quotes that were posted with a privacy of Public?

**Comment by TomatDividedBy0** [comment]
Yeah, it's separate because it'd be something where if I looked up "To Kill a Mockingbird" I'd get all the public quotes, irrespective of if I follow the people who posted them.

**Comment by soshial** [comment]
I would also appreciate a possibility to share a quote publicly and send a link to a friend, for example.

**Comment by mouse-reeve** [comment]
> I would also appreciate a possibility to share a quote publicly and send a link to a friend, for example.

This is already possible! You can link directly to any post by clicking the timestamp on it. For example: https://bookwyrm.social/user/mouse/quotation/1851297

It might be helpful to add "share" options in the little three-dot post menu, but that is a different feature request than this one :)

**Comment by soshial** [comment]
Thank you for the tip and for such a great service with great UI, @mouse-reeve! The principles of BookWyrm and clean and uncluttered UI really motivated me to read.

Still, showing a public quote publicly, visible to non-logged users would be nice :)

**Comment by xuv** [comment]
I feel this is related, so don’t want to open a new issue about this.
As @mouse-reeve it’s possible to link to a direct quoted bit left by a user: For example: https://bookwyrm.social/user/mouse/quotation/1851297

I would love to be able to access all the quoted bits that a user made. Taking the example above, something like For example: https://bookwyrm.social/user/mouse/quotations could be great. 

Right now, I can see all the reviews and comments from a user, https://bookwyrm.social/user/mouse/reviews-comments but it would be lovely to see all the quoted bits too in its own page.
